src/rev_ai/apiclient.py

```diff
@@ -322,8 +341,29 @@ def get_account(self):
 
         :raises: HTTPError
         """
-        url = urljoin(self.base_url, 'account')
-        response = self.session.get(url)
-        response.raise_for_status()
+        response = self._HTTPHandler(
+                      "GET",
+                      urljoin(self.base_url, 'account')
+        )
 
         return Account.from_json(response.json())
+
+    def _HTTPHandler(self, method, url, **kwargs):
```

Comment:
This is still named as if it's a class (a noun). It should be named like a method (a verb). Something like `make_http_request`